jax/numpy/lax_numpy.py

```diff
@@ -3089,6 +3089,18 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.convolve)
+def convolve(x, y, mode='full'):
+  x_jax = lax.reshape(x,(1,1,len(x)))
+  y_jax = flip(lax.reshape(y,(1,1,len(y))),2)
+  if mode == 'full':
+    return ravel(lax.conv_general_dilated(x_jax,y_jax,[1],[(len(x)-1,len(x)-1)]))
```

Comment:
Is this the right padding? I don't think so: I would have assumed the padding would be related to the size of `y`, not the size of `x`. Certainly this doesn't seem to produce the right output when `x` and `y` don't have the same size.